cid/helpers/quicksight/dashboard.py

```diff
@@ -101,22 +101,38 @@ def status(self) -> str:
         return self._status
 
     def display_status(self) -> None:
+          
         print('\nDashboard status:')
         print(f"  Name (id): {self.name} ({self.id})")
         print(f"  Status: {self.status}")
         print(f"  Health: {'healthy' if self.health else 'unhealthy'}")
         if self.status_detail:
             print(f"  Status detail: {self.status_detail}")
+            
+        cid_version = None
+        cid_version_latest = None
+        try:
+            cid_version = self.deployedTemplate.cid_version
+        except ValueError:
+            cid_version = "UNKNOWN"
+        
+        try:
+            cid_version_latest = self.sourceTemplate.cid_version if isinstance(self.sourceTemplate, CidQsTemplate) else "UNKNOWN"
+        except ValueError:
+            cid_version_latest = "UNKNOWN"
         if self.latest:
             print(f"  Version: {self.deployed_version}")
         else:
             print(f"  Version (deployed, latest): {self.deployed_version}, {self.latest_version}")
+            
+        print(f"  CID Version (deployed, latest): {cid_version}, {cid_version_latest}")
```

Comment:
not clear what is CID version here. probably can be rephrased:

print(f" Deployed Version={cid_version}->Latest TemplateVersion= {cid_version_latest}")